docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,93 @@
+
+3. Select the `Permissions` tab and check the `Execute` box.
+
+![AppImage Execute Permission](/images/create-bootable-usb-linux/appimage-execute.png)
+
+4. Double click the Popsicle AppImage to launch it.
+### Flatpak
+
+1. Follow the instructions on [Flatpak.org](https://flatpak.org/setup/) if Flatpak is not installed in your Linux distribution.
+
+2. Open a Terminal session and enter this command: 
+
+```bash
+flatpak install flathub com.system76.Popsicle
+```
+
+3. Enter this command to launch Popsicle: 
+
+```bash 
+flatpak run com.system76.Popsicle
+```
+## Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](/Getting-Started/Create-bootable-media/create-bootable-usb.html#choose-a-pop_os-image) for information about each version. 
+
+Take note of the hash value for your chosen ISO. Use this value to verify the integrity of the ISO file in step four of the installation process.
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+## Create the Bootable USB
+
+Flash your USB media with the Pop!_OS.iso image. Note that this process will completely erase all previous data on the USB storage device.
+
+1. Launch Popsicle. The app may display as USB Flasher in search results.
+
+![Choose Image](/images/create-bootable-usb-linux/launch-popsicle-app.png)
+
+2. Click `Choose Image`.
+
+![Launch Popsicle](/images/create-bootable-usb-linux/choose-image.png)
+
+3. Navigate to local directory with your Pop!_OS.iso file and select it.
+
+![Find Image](/images/create-bootable-usb-linux/find-image.png)
+
+4. Click the drop down menu next to `Hash` and choose `SHA256`, then click `Check`. Compare the hash result to the value you saw when selecting your Pop!\_OS.iso file.
+
+![Verify ISO](/images/create-bootable-usb-linux/verify-iso.png)
+
+4. Click `Next`. 
+
+5. Select the USB installation media. Click `Next`.
+
+![Select Media](/images/create-bootable-usb-linux/select-media.png)
+
+6. Enter your system password if prompted, then wait for the installation to complete.
+
+![Flashing Process](/images/create-bootable-usb-linux/flashing-process.png)
+
+7. Click `Done` when the process completes. 
+
+## Next Steps
+
+Use your bootable Pop!_OS USB to demo, install, or recover a current Pop!_OS installation.
+
+### Install Pop!_OS
+
+Power off the target computer and insert the bootable USB. Power on the computer and enter the boot device menu selection for your BIOS or UEFI system. Follow the steps in the [Standard Installation](/Getting-Started/Installation/installation.md) section.
+
+>**Note**: Systems with Secure Boot enabled must disable this feature before installing Pop!_OS. See [Microsoft's official documentation](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/disabling-secure-boot?view=windows-10) for this procedure.
```

Comment:
This link was removed from the other page, was it supposed to be left here? Do we need to address Secure Boot here on the USB page in addition to the install/demo pages?

Reply:
I included it here in case someone skipped standard installation steps, but on review I don't think it needs to be in this chapter. It's already in the download pop-up on pop.system76.com, and if users run into this issue they can reference the standard install instructions for this info.